src/main/java/uk/gov/companieshouse/insolvency/data/controller/InsolvencyController.java

```diff
@@ -33,7 +33,7 @@
             @PathVariable("company_number") String companyNumber,
             @RequestBody InsolvencyRequest requestBody
```

Comment:
I dont think we need InsolvencyRequest wrapper class is required? lets discuss offline